sos/policies/__init__.py

```diff
@@ -835,8 +843,15 @@ class LinuxPolicy(Policy):
     vendor = "None"
     PATH = "/bin:/sbin:/usr/bin:/usr/sbin"
     init = None
-
+    _upload_url = None
+    _upload_directory = '/'
+    _upload_user = None
+    _upload_password = None
+    _use_https_streaming = False
     _preferred_hash_name = None
+    upload_url = None
+    upload_user = None
+    upload_password = None
```

Comment:
For better understanding, I would love to have a comment why there are "duplicate" variables like `upload_url` / `_upload_url`.